src/Storefront/Subscriber/FilterWhitelistSubscriber.php

```diff
@@ -14,6 +14,7 @@
 class FilterWhitelistSubscriber implements EventSubscriberInterface
 {
     public const CUSTOM_FILTERS_GROUP_IDS = 'custom_filters_group_ids';
+    private const WHITELIST_FILTER_PRIORITY = 200;
```

Comment:
How about we whitelist `*Subscriber` classes from Magic Number Detector? IMO moving the priorities to constants will do more harm than good.

Reply:
Sure, no problem to exclude them.

Reply:
@bartosz-zasada updated by ccb38dc